lib/features/photo/remote_photo_repository.dart

```diff
@@ -110,6 +112,23 @@ class PhotoRepository {
     }
   }
 
+  Future<void> registerPhotoData({
+    required String userId,
+    required UnionTimestamp registerShotAt,
```

Comment:
提案していた`registerShotAt`というのはメソッド名だったのですが、実際に作成するドキュメントはphotoなのでregisterPhotoDataでも良いと思います！
なので、引数はそのままの`shotAt`で良いかなと思います！
```suggestion
    required UnionTimestamp shotAt,
```